app/uploaders/veteran_facing_forms_remediation_uploader.rb

```diff
@@ -4,42 +4,62 @@ class VeteranFacingFormsRemediationUploader < CarrierWave::Uploader::Base
   include SetAWSConfig
   include UploaderVirusScan
 
+  class << self
+    # TODO: update this to vff specific S3 bucket once it has been created
+    # e.g. Settings.vff_simple_forms.s3
+    def s3_settings
+      Settings.reports.aws
+    end
+
+    def new_s3_resource
+      Aws::S3::Resource.new(
+        region: s3_settings.region,
+        access_key_id: s3_settings.aws_access_key_id,
+        secret_access_key: s3_settings.aws_secret_access_key
+      )
+    end
+
+    def get_s3_link(file_path)
+      new_s3_resource.bucket(s3_settings.bucket)
+                     .object(file_path)
+                     .presigned_url(:get, expires_in: 30.minutes.to_i)
+    end
+  end
+
   def size_range
-    (1.byte)...(100_000_000.bytes)
+    (1.byte)..(100.megabytes)
   end
 
-  # All the same files allowed by benefits intake, with the
-  # addition of json for metadata and csv for manifest
+  # Allowed file types, including those specific to benefits intake
   def extension_allowlist
-    %w[bmp csv gif jpeg jpg json pdf png tif tiff txt]
+    %w[bmp csv gif jpeg jpg json pdf png tif tiff txt zip]
   end
 
   def initialize(benefits_intake_uuid, directory)
     raise 'The benefits_intake_uuid is missing.' if benefits_intake_uuid.blank?
+    raise 'The s3 directory is missing.' if directory.blank?
 
-    super
     @benefits_intake_uuid = benefits_intake_uuid
     @directory = directory
 
+    super()
```

Comment:
Ooh is this what was missing before?

Reply:
This is definitely something that was missing before. I THINK this wasn't an issue, just syntactically incorrect originally.